src/controllers/authController.js

```diff
@@ -29,24 +29,36 @@ exports.register = async (req, res) => {
   }
 };
 
+// @route GET api/auth/login
+// @desc Render login form
+// @access Public
+exports.getLoginPage = (req, res) => {
+  res.status(200).render('login');
+}
+
 // @route POST api/auth/login
 // @desc Login user and return JWT token
 // @access Public
 exports.login = async (req, res) => {
   try {
-    const { email, password } = req.body;
-    const user = await User.findOne({ email });
+    const { username, password } = req.body;
+    const user = await User.findOne({ username }).populate({
+      path: "bookedTours",
+      populate: {path: "tour"}
+    });
+    // console.log(user)
     if (!user)
       return res.status(401).json({
         msg:
-          "The email address " +
+          "The username address " +
           email +
-          " is not associated with any account. Double-check your email address and try again.",
+          " is not associated with any account. Double-check your username and try again.",
       });
 
     //validate password
     // if (!user.comparePassword(password)) return res.status(401).json({ message: "Invalid email or password" });
     user.authenticate(password, (error, foundUser, passwordError) => {
+      console.log(user.bookedTours[0].tour.title);
       if (error) res.status(500).json({ message: error.message });
```

Comment:
is there an error page from the frontend yet? If there is, use that page to render all error messages instead of returning json